### DISCUSSION TITLE
How to remove blur from search dialog?

### DISCUSSION THREAD

**Discussion by Dhruv-Garg79** [chrismwilliams/astro-theme-cactus#228] [General]
I am trying to make backgroud of search dialogue opaque like in example website, but I am unable to do so. Tried changing these classes - `bg-bgColor shadow backdrop:backdrop-blur`, but unable to do it

**Comment by chrismwilliams** [comment]
You're almost there, if you remove `backdrop:backdrop-blur` class from `<dialog>` it will remove the blur from the [::backdrop](https://developer.mozilla.org/en-US/docs/Web/CSS/::backdrop) pseudo element, which is what fills the viewport.

`bg-bgColor` and `shadow` are targeting just the `<dialog>` element.
